_site/all_events.html

```diff
@@ -41,7 +41,7 @@
   <button class="w3-bar-item w3-button w3-large"
   onclick="w3_close()"><i class="fa fa-firefox"></i> Mozilla</button>
   <a href="index.html" class="w3-bar-item w3-button"><i class="fa fa-home"></i>   Home</a>
-  <a href="#" class="w3-bar-item w3-button"><i class="fa fa-line-chart"></i>   Dashboard</a>
+  <a href="https://anamikad.github.io/" class="w3-bar-item w3-button"><i class="fa fa-line-chart"></i>   Dashboard</a>
```

Comment:
Link works great, but can we link to a mozillascience/ domain on github once we're done with testing. I just worry that the repo will get lost if it's not in our typical organization queue; you're welcome to keep it at anamikad.github.io but you might want to use that for your own website eventually, or another project, so it would be cool to keep it archived at a url that mozillascience maintaints.